streamparser.py

```diff
@@ -158,7 +159,15 @@ def __init__(self, lexical_unit):  # type: (str) -> None
         self.lexical_unit = lexical_unit
 
         cohort = re.split(r'(?<!\\)/', lexical_unit)
-        self.wordform = cohort[0]
+
+        if("]]^" in cohort[0]):
+            split_form = cohort[0].split("]]^")
+            self.wordbound_blank = split_form[0] + "]]"
```

Comment:
If the split is limited, you can name each element, i.e.
`wordbound_blank, wordform = cohort[0].split(...)`